ansible/roles/docker/docker-image/tasks/main.yml

```diff
@@ -7,4 +7,4 @@
 
 - name: Pulling the image
   docker_image:
-    name: "{{image}}:{{tag}}"
+    name: "{{image}}"
```

Comment:
how you could could pull image without tag name?

Reply:
In the current docker one-liner, we are used to specifying the image (or -i) with e.g. shadowrobot/dexterous-hand:kinetic-v1.0.15 (image and tag together). I evaluated options: A) require user to specify image and tag separately (different how we are currently doing it) or B) keep current pattern in docker one-liner to take "image" to be actually image:tag combination. That's what I'm going here. {{ image }} is actually {{ shadowrobot/dexterous-hand:kinetic-v1.0.5 }}

Reply:
https://github.com/shadow-robot/sr-build-tools/blob/master/docker/launch.sh#L650